tox.ini

```diff
@@ -53,19 +55,26 @@ pass_env =
     CODECOV_*
     DD_*
     STPSF_PATH
+set_env =
+    devdeps: UV_INDEX = https://pypi.anaconda.org/liberfa/simple https://pypi.anaconda.org/astropy/simple https://pypi.anaconda.org/scientific-python-nightly-wheels/simple
+    devdeps: UV_INDEX_STRATEGY = unsafe-any-match
 extras =
     test
     alldeps: all
+uv_resolution =
+    oldestdeps: lowest-direct
 deps =
     xdist: pytest-xdist
     ddtrace: ddtrace
-    oldestdeps: minimum_dependencies
+    stdevdeps,devdeps: -r requirements-dev-st.txt
+    devdeps: -r requirements-dev-thirdparty.txt
+    sdpdeps: -r requirements-sdp.txt
+    oldestdeps: minimum_dependencies>=0.1.0
+    oldestdeps: setuptools>=69  # for pkg_resources for pysiaf
 commands_pre =
     oldestdeps: minimum_dependencies romancal --filename requirements-min.txt
-    oldestdeps: pip install -r requirements-min.txt
-    stdevdeps: pip install -r requirements-dev-st.txt -U --upgrade-strategy eager
-    devdeps: pip install -I -r requirements-dev-thirdparty.txt -r requirements-dev-st.txt -U --upgrade-strategy eager
-    pip list
+    oldestdeps: uv pip install -r requirements-min.txt
```

Comment:
is this still necessary if we're using `lowest-direct` now?
```suggestion
```

Reply:
it still is required according to @braingram https://github.com/spacetelescope/roman_datamodels/pull/429#pullrequestreview-2458721475